README.md

```diff
@@ -6,7 +6,91 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
 
 If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
 
+## TSC Scope of Responsibility
+
+**The TSC is responsible for Node.js Core project, a number of projects depended upon by Node.js Core, and a number of projects adjacent to Node.js Core.**
+
+The TSC exercises autonomy in managing its responsibilities and seeks agreement from the Node.js Foundation Board on any change to the scope of those responsibilities as defined below.
+
+### Node.js Core
+
+Node.js Core is defined as the code project existing in the repository located at <https://github.com/nodejs/node> which the `node` executable and associated packages are built from, _minus_ the vendored projects included in the codebase, specifically those located in the [deps](https://github.com/nodejs/node/tree/master/deps) and [tools](https://github.com/nodejs/node/tree/master/tools) directories that are copied in from externally maintained repositories.
+
+### Dependent Projects
+
+Additional projects fall within the TSC's scope of responsibility which are dependencies of Node.js Core in that they are **required** to plan, test, build, document and ship Node.js releases.
+
+These projects are located in the following repositories:
+
+* https://github.com/nodejs/build — administration delegated to the Build Working Group
+* https://github.com/nodejs/citgm — administration delegated to the Core Technical Committee
```

Comment:
Ambiguous so I said CTC, could be Build, could be Testing, could even be LTS or it a whole new Smoke Testing group!

@TheAlphaNerd what say you?